charts/kube-prometheus-stack/templates/alertmanager/ingress.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and .Values.alertmanager.enabled .Values.alertmanager.ingress.enabled }}
-{{- $pathType := .Values.alertmanager.ingress.pathType | default "" }}
+{{- $pathType := .Values.alertmanager.ingress.pathType | default "ImplementationSpecific" }}
```

Comment:
Maybe the empty string is possible. The evaluation would have to be adjusted, but I think using the default is a good practice especially since the API is stable.